soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -414,16 +414,13 @@ void FileChoose_UpdateMainMenu(GameState* thisx) {
         fileSelectSpoilerFileLoaded = false;
     }
 
-    if ((CVar_GetS32("gNewFileDropped", 0) != 0) ||
```

Comment:
nitpick: looks like we're just making formatting changes here.. would not have to be part of the PR